events/elastic-search.md

```diff
@@ -0,0 +1,17 @@
+# ElasticSearch, You Know For Search
+
+## What was the event about?
+
+Aravind Putrevu, a developer advocate for Elastic, gave a talk on the rising usage of ElasticSearch and also presented a demo
+on how easy it is to setup and run.
+
+## Archives
+
+### 2019
+
+Facebook Teaser - https://www.facebook.com/kossiitkgp/posts/2142807422462913
+Facebook Announcement - https://www.facebook.com/kossiitkgp/posts/2152998678110454
+Facebook Event - https://www.facebook.com/events/2360503774228061/
+Event Slides - https://noti.st/aravindputrevu/3ngeiZ/elasticsearch-you-know-for-search
+
```

Comment:
```suggestion
Facebook Teaser - https://www.facebook.com/kossiitkgp/posts/2142807422462913

Facebook Announcement - https://www.facebook.com/kossiitkgp/posts/2152998678110454

Facebook Event - https://www.facebook.com/events/2360503774228061/

Event Slides - https://noti.st/aravindputrevu/3ngeiZ/elasticsearch-you-know-for-search
```

For each to be on different lines in the rendered version - https://github.com/kossiitkgp/docs/blob/3876356e0f2a2096c8ba7a2c2d56fc934caa1f95/events/elastic-search.md